cmake/pcl_find_sse.cmake

```diff
@@ -169,27 +169,27 @@ macro(PCL_CHECK_FOR_SSE)
 
     if(CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX OR CMAKE_COMPILER_IS_CLANG)
         if(HAVE_SSE4_2_EXTENSIONS)
-            SET(SSE_FLAGS "${SSE_FLAGS} -msse4.2 -mfpmath=sse")
+            list(APPEND SSE_FLAGS "-msse4.2" "-mfpmath=sse")
         elseif(HAVE_SSE4_1_EXTENSIONS)
-            SET(SSE_FLAGS "${SSE_FLAGS} -msse4.1 -mfpmath=sse")
+            list(APPEND SSE_FLAGS "-msse4.1" "-mfpmath=sse")
         elseif(HAVE_SSSE3_EXTENSIONS)
-            SET(SSE_FLAGS "${SSE_FLAGS} -mssse3 -mfpmath=sse")
+            list(APPEND SSE_FLAGS "-mssse3" "-mfpmath=sse")
         elseif(HAVE_SSE3_EXTENSIONS)
-            SET(SSE_FLAGS "${SSE_FLAGS} -msse3 -mfpmath=sse")
+            list(APPEND SSE_FLAGS "-msse3" "-mfpmath=sse")
         elseif(HAVE_SSE2_EXTENSIONS)
-            SET(SSE_FLAGS "${SSE_FLAGS} -msse2 -mfpmath=sse")
+            list(APPEND SSE_FLAGS "-msse2" "-mfpmath=sse")
         elseif(HAVE_SSE_EXTENSIONS)
-            SET(SSE_FLAGS "${SSE_FLAGS} -msse -mfpmath=sse")
+            list(APPEND SSE_FLAGS "-msse" "-mfpmath=sse")
         else()
             # Setting -ffloat-store to alleviate 32bit vs 64bit discrepancies on non-SSE
             # platforms.
-            set(SSE_FLAGS "-ffloat-store")
+            list(APPEND SSE_FLAGS "-ffloat-store")
         endif()
     elseif(MSVC AND NOT CMAKE_CL_64)
         if(HAVE_SSE2_EXTENSIONS)
-            SET(SSE_FLAGS "${SSE_FLAGS} /arch:SSE2")
+            list(APPEND SSE_FLAGS "/arch:SSE2")
         elseif(HAVE_SSE_EXTENSIONS)
-            SET(SSE_FLAGS "${SSE_FLAGS} /arch:SSE")
```

Comment:
Is this on purpose that `SSE` became `SSE3`?

Reply:
Bad copy paste.